src/test/test-sizeof.c

```diff
@@ -28,6 +29,14 @@ DISABLE_WARNING_TYPE_LIMITS;
                (t)-1 < (t)0 ? ", signed" : ", unsigned",                \
                alignof(t))
 
+#define check_no_sign(t, size)                  \
+        info_no_sign(t);                        \
+        assert_se(sizeof(t) == size)
```

Comment:
out of paranoia i'd still prefer the usual `do {`, `} while(false)` wrap for macros like this